openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -225,7 +233,8 @@ private Optional<OpenAPI> getOpenAPIContract(Path ballerinaFilePath, Path openAP
                 relativePath = Paths.get(openapiContract.getCanonicalPath());
             }
             if (Files.exists(relativePath)) {
-                return Optional.of(normalizeOpenAPI(relativePath, operationIdValidationRequired(toolContext)));
+                return Optional.of(normalizeOpenAPI(relativePath, operationIdValidationRequired(toolContext),
+                        isSanitized));
```

Comment:
The information is already present on the tool context right? Can't we reuse that?